packages/cli/src/utils/deepMerge.ts

```diff
@@ -28,11 +28,16 @@ function mergeRecursively(
   path: string[] = [],
 ) {
   for (const key of Object.keys(source)) {
-    if (key === '__proto__' || key === 'constructor' || key === 'prototype') {
+    // JSON.parse can create objects with __proto__ as an own property.
```

Comment:
constructor and prototype didn't need to be skipped as they won't be included in Object.